services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/modules/db/repositories/osparc.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import sqlalchemy as sa
+from models_library.projects import ProjectID
+from simcore_postgres_database.models.projects import projects
+from simcore_postgres_database.models.users import users
+
+from ._base import BaseRepository
+
+_logger = logging.getLogger(__name__)
+
+
+class OSparcRepository(BaseRepository):
+    async def get_user_email(self, user_id: int) -> str | None:
+        async with self.db_engine.connect() as conn:
+            result = await conn.execute(
+                sa.select(users.c.email).where(users.c.id == user_id)
+            )
+            data = result.first()
+            output: str | None = data[0] if data else None
+            return output
+
+    async def get_project_name_and_workbench(
+        self, project_uuid: ProjectID
+    ) -> tuple[str, dict] | None:
+        async with self.db_engine.connect() as conn:
+            result = await conn.execute(
+                sa.select(projects.c.name, projects.c.workbench).where(
+                    projects.c.uuid == f"{project_uuid}"
+                )
+            )
+            output = result.first()
```

Comment:
MINOR: this is just a matter of style but i think that it is easier to read if you are consistent in the name of the variables. Look these two functions. You use `output` and `data` for the same thing.

TIP: use `result` for `execute` returned value and ``row`` for the returned value of `result`